### ISSUE TITLE
training is killed

### ISSUE THREAD

**Issue by zhaojinqiang** [norybaby/poet#4]
01:20:32 INFO:Start training
2018-12-15 13:20:32.416065: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
Killed

what's wrong..

**Comment by xxllp** [comment]
是不是版本的问题

**Comment by zhaojinqiang** [comment]
Writing to /usr/local/poet-master/output_poem/tensorboard_log/1555315332

============================================================
All final and intermediate outputs will be stored in output_poem/
============================================================

04:02:12 INFO:args are:
Namespace(batch_size=16, best_model='', best_valid_ppl=inf, cell_type='lstm', data_path='./data/poem/', debug=False, dropout=0.0, embedding_size=128, encoding='utf-8', hidden_size=128, init_dir='', init_model='', input_dropout=0.0, learning_rate=0.005, max_grad_norm=5.0, num_epochs=5, num_layers=2, num_unrollings=64, output_dir='output_poem', progress_freq=100, save_best_model='output_poem/best_model/model', save_model='output_poem/save_model/model', tb_log_dir='/usr/local/poet-master/output_poem/tensorboard_log/1555315332', test=False, train_frac=0.9, valid_frac=0.05, verbose=0)
04:02:12 INFO:Parameters are:
{
    "batch_size": 16,
    "cell_type": "lstm",
    "dropout": 0.0,
    "embedding_size": 128,
    "hidden_size": 128,
    "input_dropout": 0.0,
    "learning_rate": 0.005,
    "max_grad_norm": 5.0,
    "num_layers": 2,
    "num_unrollings": 64
}

tensor_file:./data/poem/poem_ids.txt
Loading dataset from ./data/poem/poem_ids.txt
file maxSeqLen = 64
Loaded ./data/poem/:  training  samples:65235 ,validationSamples:3837,testingSamples:7676
tensor_file:./data/poem/poem_ids.txt
Loading dataset from ./data/poem/poem_ids.txt
Killed


这个是什么原因呢? 谢谢.

**Comment by xxllp** [comment]
你这个不会是资源不足吧
